### PR TITLE
Updates for some packages

### PR DESCRIPTION
**Pull request by renatosilva**
* Fix regression with hunspell and enchant.
* Update pidgin and libmongoose.
* Install license for graphite2.

### CODE REVIEW

**Comment by fracting** [comment]
```
error: target not found: mingw-w64-x86_64-hunspell<1.4
==> ERROR: 'pacman' failed to install missing dependencies.
```
Why this error?

**Comment by Alexpux** [comment]
because hunspell more than 1.4?

**Comment by fracting** [comment]
hunspell 1.3.4-10 is in repo:

http://mirrors.tea-ci.org/msys2/mingw/i686/mingw-w64-i686-hunspell-1.3.4-10-any.pkg.tar.xz

Is it possible a syntax error?

File: mingw-w64-enchant/PKGBUILD [hunk 1]
```diff
@@ -14,7 +14,7 @@ makedepends=("${MINGW_PACKAGE_PREFIX}-gcc" "${MINGW_PACKAGE_PREFIX}-pkg-config")
 depends=(${MINGW_PACKAGE_PREFIX}-gcc-libs
          ${MINGW_PACKAGE_PREFIX}-glib2
          #${MINGW_PACKAGE_PREFIX}-aspell
-         ${MINGW_PACKAGE_PREFIX}-hunspell)
+         ${MINGW_PACKAGE_PREFIX}-hunspell{'>=1.3','<1.4'})
```
Comment (Alexpux) [inline mingw-w64-enchant/PKGBUILD]: Wrong syntax. Also soon hunspell will be updated so what next then?

**Comment by renatosilva** [comment]
That error is expected because hunspell needs to be downgraded. Alternatively, someone can make enchant work, bring that commit back, and have enchant updated instead. The version specification is normal bash syntax.

**Comment by mati865** [comment]
I think we should revert hunspell till enchant is fixed.

**Pull request merged**